src/annotator/test/adder-test.js

```diff
@@ -367,4 +366,20 @@ describe('Adder', () => {
       );
     });
   });
+
+  describe('#hides', () => {
```

Comment:
```suggestion
  describe('#hide', () => {
```

Should match the member name exactly